otel/conf.go

```diff
@@ -166,6 +166,8 @@ windowsperfcounters/mssql_{{.MSSQLID}}:
           - system.network.dropped
           - system.filesystem.inodes.usage
           - system.paging.faults
+          - system.disk.operation_time
+          - system.processes.count
```

Comment:
Should this line be reflected in the other golden files as well?

Reply:
Yes, it should. Forgot to regenerate the golden files again.

Reply:
The good thing is that our tests caught it as well, which is what they are designed for :D